workshop/content/docs/introduction.md

```diff
@@ -20,10 +20,11 @@ Your text editor needs to be able to edit configuration files in **plain text**.
 not limited to:
 
 * Notepad or Notepad++ (Windows)
-* Sublime Text (Linux/Mac/Windows)
-* TextEdit (Mac)
-* Visual Studio Code (Linux/Mac/Windows)
-* `/usr/bin/vim` (aka [Tom's](https://twitter.com/tomkralidis) favourite :))
+* Sublime Text (Linux/Mac/Windows): Angelos
+* TextEdit (Mac): Just
```

Comment:
You can leave TextEdit out, instead: [IntelliJ IDEA](https://www.jetbrains.com/idea/) (Mac).

(is my main editor for really everything: Python, JS, JSON, XML, YAML, Bash, SQL, HTML, CSS it has syntax-aware  support for all languages.)